### PR TITLE
added snapid + request schemas

### PR DESCRIPTION
**Pull request by shanejonas**

### CODE REVIEW

**Review by BelfordZ — APPROVED** [review]

File: openrpc.json [hunk 1]
```diff
@@ -766,10 +767,20 @@
...
               "request": {
-                "description": "The JSON-RPC request object to send to the invoked Snap."
+                "description": "The JSON-RPC request object to send to the invoked Snap.",
+                "type": "object",
+                "properties": {
+                  "method": {
+                    "type": "string"
+                  },
+                  "params": {
+                    "type": "string"
```
Comment (BelfordZ) [inline openrpc.json:781]: should this be left as 'any' ? I guess our interactive docs dont really know what to do with it in that case though...

Comment (shanejonas) [inline openrpc.json:781]:
```suggestion
                    "type": "object"
```

File: openrpc.json [hunk 2]
```diff
@@ -828,10 +839,20 @@
...
```
Comment (shanejonas) [inline openrpc.json:853]:
```suggestion
                    "type": "object"
```

File: openrpc.json (hunk 1 above)
Comment (shanejonas) [reply openrpc.json:781]: should really be `oneOf` object or array

Comment (alexandratran) [reply openrpc.json:781]: Since `oneOf` is not supported I think (https://github.com/open-rpc/spec/issues/278), we'll just go with object for now?

**Review by alexandratran — APPROVED** [review]

**Pull request merged**